src/widgets/ReviewsBlock/ui/ReviewsBlock/ReviewsBlock.tsx

```diff
@@ -27,41 +26,83 @@ export type Props = {
  * @param {string} linkPath - адрес ссылки
  */
 const ReviewsBlock: FC<Props> = props => {
-  const { title, linkText = '', linkPath = '' } = props
+  const { title, linkText = '', linkPath = '#' } = props
   const linkTextStyle = styles.link
 
   const dispatch = useAppDispatch()
-  const reviews: StoreReviewData[] = useSelector(getStoreReviewsSelector)
+  const reviews = useSelector((store: StateSchema) => store.feedbacks)
+  const listRef = useRef<HTMLDivElement>(null)
+  const startX = useRef(0)
+  const startScrollLeft = useRef(0)
+  const isDragging = useRef(false)
 
   useEffect(() => {
-    //TODO реализовать пагинацию, временно отображать 1-ую страницу
-    dispatch(getStoreReviews(1))
+    dispatch(getFirstFeedbacks())
+
+    dispatch(getAverageMark())
   }, [])
 
+  const handleMouseDown = (e: MouseEvent<HTMLDivElement, globalThis.MouseEvent>) => {
+    startX.current = e.clientX
+    if (listRef.current) {
```

Comment:
когда проверяем ref, мы проверяем сначала наличие самого реф, вот так
```suggestion
    if (listRef && listRef.current) {
```